docs/installing.rst

```diff
@@ -120,51 +126,91 @@ Two virtualization providers have been tested with the ADB.
       # Set libvirtd to start automatically on system boot
       $ sudo systemctl enable libvirtd
 
+    This would install both Vagrant and Libvirt. 
+
 * CentOS
 
-  Vagrant packages are not available directly in CentOS core. However they are
+  Vagrant packages are not available directly in CentOS core. However, they are
   available through official CentOS `Software Collections
   <http://softwarecollections.org>`_ builds.
 
-  Here are the commands to get Vagrant in CentOS with `libvirt` support::
+  Here are the commands to get Vagrant in CentOS::
 
     $ sudo yum -y install centos-release-scl
     $ sudo yum -y install sclo-vagrant1
     $ sudo scl enable sclo-vagrant1 bash
+  
+  To add `libvirt` support use this::
 
     # Start libvirtd
     $ sudo systemctl start libvirtd
 
     # Set libvirtd to start automatically on system boot
     $ sudo systemctl enable libvirtd
 
+-----------------------------------------------------
+3. Install the vagrant-service-manager Vagrant plugin
+-----------------------------------------------------
+
+The `vagrant-service-manager`_ plugin can be installed using::
+
+    vagrant plugin install vagrant-service-manager
+
+.. _vagrant-service-manager: https://github.com/projectatomic/vagrant-service-manager
+
 -------------------
-3. Download the ADB
+4. Download the ADB
 -------------------
 
 There are two ways to download the ADB.  You can have ``vagrant`` do it for you
 the first time you install it or you can download it manually.
 
-* ``vagrant`` Initiated Download
+* Vagrant Initiated Download
 
-  The image is available at https://atlas.hashicorp.com/projectatomic/boxes/adb.
   The ``vagrant`` program is capable of downloading the box the first time it is
   needed. This happens when you first initialize a new vagrant environment.
 
-  If you wish to use a project provided vagrant file you should first get the
-  Vagrantfile as directed in `Using the Atomic Developer Bundle`_ in
-  the *Using Custom Vagrantfiles for Specific Use Cases* section.
+  The ADB project provides a few customised Vagrantfiles, which can auto download the ADB, in the `Using Custom Vagrantfiles for Specific Use Cases`_ section of the `Using the Atomic Developer Bundle`_ document. The ADB images are cataloged at `atlas.hashicorp.com`_.
```

Comment:
The ADB project provides customized Vagrantfiles, which will download the ADB and automatically set up specific container development environments.  They are listed below and more details are available, in the `Using Custom Vagrantfiles for Specific Use Cases`_ section of the `Using the Atomic Developer Bundle`_ document.